starter-code/src/array-functions.js

```diff
@@ -4,30 +4,49 @@ var ArrayFunctions = function (){}
 // ArrayFunctions.printSpecial([12, 33, 144, 122])
 // 12 -- 33 -- 144 -- 122
 
+// ArrayFunctions.prototype.printSpecial = function (array) {
+//   return array.join(" --- ");
+// };
+
 ArrayFunctions.prototype.printSpecial = function (array) {
-  return array.join(" --- ");
+  return array.join(" --- ")
 };
 
+
+
 // 2) Define a function that takes an array with numbers and returns another array where each element contains double each element in the array
 // ArrayFunctions.doubleMyArray([10, 20, 35, 12])
 // [20, 40, 70, 24]
 
-ArrayFunctions.prototype.doubleArray = function(array){
-  return array.map(function(number){
-    return number * 2;
-  });
+// ArrayFunctions.prototype.doubleArray = function(array){
+//   return array.map(function(number){
+//     return number * 2;
+//   });
+// };
+
+ArrayFunctions.prototype.doubleArray = function (array) {
+  return array.map(function(x){
```

Comment:
A better way to convert this into ES6 is to do this
`return array.map( x => x * 2);`